app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeakFragment.java

```diff
@@ -90,6 +94,8 @@ public void callback(ArrayList voiceResult)
                 voiceInput = voiceResult;
                 speechTextDisplay.setText(voiceInput.get(0).toString());
                 isListening = false;
+                //spokenString = "why are my hands so smelly?";
+                new GetWordsAsync(voiceInput.get(0).toString(), importantWords, getActivity().getApplicationContext(), commentText).execute();
```

Comment:
why are you passing in `importantWords` when it's an empty string?
also, i feel like it'd be better if you just had a callback that brought back a response in this fragment so the string would be easier to put to speech